### ISSUE TITLE
Another problem, probably related to docker

### ISSUE THREAD

**Issue by yg10** [tezos/tezos#101]
```
./alphanet.sh rpc call blocks with  '{"include_ops": true}'
Fatal error:
  Command not found
Try `-help` for a list of options and commands.

```
while from within container the command
` tezos-client rpc call blocks with  '{"include_ops": true}'
`
works and returns a list of blocks.

I am blaming docker for misquoting shell strings. See #97

**Comment by MiloDavis** [comment]
This is the same problem as #97.

**Comment by venator** [comment]
Fixed with commit b1e5c719da044d3344af2109b5e6fcb153408d31
